### PR TITLE
Update use-arc-exceptions-to-mark-trusted-arc-senders.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the Authenticated Received Chain (ARC) in Microsoft 365 Defender for Office section

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9356

### CODE REVIEW

File: microsoft-365/security/office-365-security/use-arc-exceptions-to-mark-trusted-arc-senders.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Email authentication mechanisms like [SPF](set-up-spf-in-office-365-to-help-prev
 
 ## Authenticated Received Chain (ARC) in Microsoft 365 Defender for Office
 
-Services that modify content during transport of the message before delivery to your organization, can invalidate the DKIM email signature and affect the authentication of the message. When these intermediary services, perform such actions, they can use ARC can be used to provide details of the original authentication before the modifications occurred, which your organization can then trust to help with authenticating the message.  
+Services that modify content during transport of the message before delivery to your organization, can invalidate the DKIM email signature and affect the authentication of the message. When these intermediary services, perform such actions, they can use ARC to provide details of the original authentication before the modifications occurred, which your organization can then trust to help with authenticating the message.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-arc-exceptions-to-mark-trusted-arc-senders.md]:
```suggestion
Services that modify content during transport of the message before delivery to your organization can invalidate the DKIM email signature and affect the authentication of the message. When these intermediary services perform such actions, they can use ARC to provide details of the original authentication before the modifications occurred, which your organization can then trust to help with authenticating the message.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy recview.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Pull request merged**